integration-test/src/test/java/com/tietoevry/quarkus/resteasy/problem/JaxRsMappersIT.java

```diff
@@ -50,23 +48,11 @@ void webApplicationExceptionShouldReturnHeaders() {
                 .log().all()
                 .statusCode(status)
                 .header(HttpHeaders.RETRY_AFTER, equalTo("120"))
-                .body("title", equalTo(Status.valueOf(status).getReasonPhrase()))
+                .body("title", equalTo(Response.Status.fromStatusCode(status).getReasonPhrase()))
                 .body("status", equalTo(status))
                 .body("stacktrace", nullValue());
     }
 
-    @Test
```

Comment:
Why it is gone?

Reply:
It was moved to separate ZalandoProblemMapperIT to make zalando stuff even more separated from everything.